antsibull/build_ansible_commands.py

```diff
@@ -19,21 +18,27 @@
 import sh
 from jinja2 import Template
 from packaging.version import Version as PypiVer
+from semantic_version import Version as SemVer
```

Comment:
If this is only used by type checking code, I'd prefer to put it in a conditional:

``` python
if t.TYPE_CHECKING:
    from semantic_version import version as SemVer
```

That way it doesn't become a new runtime dependency for this file (Probably not a big deal for this library which is both small and doesn't lead to any circular dependencies with our code but it's a good habit that I'd like to maintain.)

Reply:
Fixed in c7b37f510703563ca2044915bf6895a9e22f5acc.